from django.utils import autoreload
import django

def execute(*args, **kwargs):
    try:
        autoreload.check_errors(django.setup)()
    except Exception as e:
        print(f"An error occurred: {e}")
        autoreload.raise_last_exception()